internal/cmdutil/cmdutil.go

```diff
@@ -62,5 +65,8 @@ func printSubCommands(cmdList []*SubCommand) {
 	log.Println("Supported sub-commands:")
 	for _, cmd := range cmdList {
 		log.Printf("\t%s - %s", cmd.Name, cmd.Short)
+		for i := range cmd.Examples {
```

Comment:
Why not `_, ex := range cmd.Examples` ?

Reply:
I would still need an index.

Reply:
No particular reason. Can be changed to the form you suggested.